tm2/pkg/crypto/keys/client/maketx.go

```diff
@@ -219,6 +220,7 @@ func ExecSignAndBroadcast(
 	io.Println("GAS USED:  ", bres.DeliverTx.GasUsed)
 	io.Println("HEIGHT:    ", bres.Height)
 	io.Println("EVENTS:    ", string(bres.DeliverTx.EncodeEvents()))
+	io.Println("TX HASH:   ", base64.StdEncoding.EncodeToString(bres.Hash))
```

Comment:
Same comment as before, should also be printed in the other case when the `DeliverTx.IsErr() == true`

Reply:
https://github.com/gnolang/gno/pull/2309/commits/9c3a0f694871ffa3625387ae10ac410d6bfebee0